assets/js/theme/product.js

```diff
@@ -46,7 +46,11 @@ export default class Product extends PageManager {
 
         const review = new Review($reviewForm);
 
-        $(document).on('opened.fndtn.reveal', () => this.reviewModal.setupFocusableElements(WRITE_REVIEW));
+        $(document).on('opened.fndtn.reveal', (event) => {
+            if ($(event.target).attr('id') === 'modal-review-form') {
+                this.reviewModal.setupFocusableElements(WRITE_REVIEW);
+            }
+        });
```

Comment:
Could we use selector argument for .on jquery method? See https://api.jquery.com/on/

Reply:
Can you please give an example? Do you mean like $( "body" )?

Reply:
`$( "#dataTable tbody" ).on( "click", "tr", function() {
  console.log( $( this ).text() );
});` from the attached link

Reply:
Done